airflow/migrations/versions/7c1bef9cb7b9_merge_heads.py

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Merge heads
+
+Revision ID: 7c1bef9cb7b9
+Revises: 3c20cacc0044, 19d8a998c007
+Create Date: 2020-06-08 14:09:06.676386
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7c1bef9cb7b9'
+down_revision = ('3c20cacc0044', '19d8a998c007')
```

Comment:
There should not be a merge migration. Please rebase your change and update the `down_revision` on your new migratin instead.

Reply:
Fixed in 648e9c2